wandb/apis/importers/wandb.py

```diff
@@ -0,0 +1,1133 @@
+
+        Optionally set:
+        - `limit` to get up to `limit` runs.
+        - `skip_ids` to ignore specific run ids
+        - `start_date` (in the format YYYY-MM-DD) to only import runs created after that date
+        """
+        filters: Dict[str, Any] = {}
+        if skip_ids is not None:
+            filters["name"] = {"$nin": skip_ids}
+        if start_date is not None:
+            filters["createdAt"] = {"$gte": start_date}
+
+        api = self.src_api
+        projects = self._projects(entity, project)
+
+        def runs():
+            for project in projects:
+                for run in api.runs(
+                    f"{project.entity}/{project.name}", filters=filters
+                ):
+                    yield WandbRun(run)
+
+        yield from itertools.islice(runs(), limit)
+
+    def import_run(self, run: WandbRun, config: Optional[ImportConfig] = None) -> None:
+        """Import one WandbRun.
+
+        Use `config` to specify alternate settings like where the run should be uploaded
+        """
+        if config is None:
+            config = ImportConfig()
+
+        settings_override = {
+            "api_key": self.dst_api_key,
+            "base_url": self.dst_base_url,
+        }
+
+        sm_config = internal.SendManagerConfig(
+            metadata=True,
+            files=True,
+            media=True,
+            code=True,
+            history=True,
+            summary=True,
+            terminal_output=True,
+        )
+
+        internal.send_run_with_send_manager(
+            run,
+            overrides=config.send_manager_overrides,
+            settings_override=settings_override,
+            config=sm_config,
+        )
+
+    def collect_artifact_sequences(
+        self,
+        entity: str,
+        project: Optional[str] = None,
+        limit: Optional[int] = None,
+    ) -> Iterable[ArtifactSequence]:
+        """Collect all of the artifact sequences from `entity`/`project`.
+
+        - If `project` is not specified, this will collect all runs from all projects
+
+        Optionally set:
+        - `limit` to get up to `limit` artifact sequences.
+        """
+        projects = self._projects(entity, project)
+
+        def sequences():
+            for project in projects:
+                # the "project" in artifact_types is actually entity/project
+                _project = f"{entity}/{project.name}"
+                for _type in self.src_api.artifact_types(_project):
+                    for collection in _type.collections():
+                        if collection.is_sequence():
+                            yield collection
+
+        # Dedupe by sequence
+        unique_sequences_map = {}
+        for seq in itertools.islice(sequences(), limit):
+            unique_sequences_map[(seq.name, seq.type)] = seq
+
+        # Dedupe artifacts within sequences
+        seen: Set[wandb.apis.public.ArtifactCollection] = set()
```

Comment:
This set holds artifacts, not artifact collections.
```suggestion
        seen: Set["Artifact"] = set()
```